nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestReplaceText.java

```diff
@@ -53,6 +54,66 @@ public TestRunner getRunner() {
         return runner;
     }
 
```

Comment:
I think the edge case is where there is a mix of expression language and other things like captures etc.
Maybe a multi-case test would be appropriate?

Reply:
I'm not sure that I follow your suggestion. I feel it is quite heavily tested at this point. Can you provide an example configuration that you think would be worthwhile testing? As in, provide a set of:
Search Value = ___
Replacement Value = ___
Replacement Strategy = ___
Sample input = ____
Expected output = ____

Reply:
Sorry, please disregard.  I literally skipped past the LITERAL part of this.